### DISCUSSION TITLE
Obtanium using Gitlab to obtain/update apk displays "forbidden"

### DISCUSSION THREAD

**Discussion by Xuiking** [ImranR98/Obtainium#1491] [Q&A]
I tried to do the process of getting the Personal Access Token and after doing it, I added it to Obtanium's settings just like what is stated in the wiki, however, it still does not work and displays "forbidden" yet again. 

Did I miss something? or did I do something wrong?

**Comment by DarkCrypt** [comment]
Grant the correct permission when creating your token.

Select scopes:

√ **read_api**
Grants read access to the API, including all groups and projects, the container registry, and the package registry.
